sst/config/config.py

```diff
@@ -0,0 +1,62 @@
+import sst
+
+DEBUG_L1 = 0
+DEBUG_MEM = 0
+DEBUG_LEVEL = 0
+
+# Define the simulation components
+cpu = sst.Component("core", "sstsimeng.simengcore")
+cpu.addParams({
+    "config_path": "<PATH TO CONFIG .YAML FILE>",
```

Comment:
Given that SST also has config files, I think it would make things 100% clear if we refer to `config` as the current framework's config (in SimEng code `config` means core model config files, in SST `config` refers to the `.py` file) and then verbosely name the other framework's config files (in SST name a SimEng-config as `simeng_config_path` for example, like-wise in SimEng for SST configs)

Reply:
I agree. Good point.

Reply:
I think so yes, although I think just `simeng_config_path` would suffice and would reduce the variable's name length